salt/tests/unit/modules/test_metalk8s_etcd.py

```diff
@@ -0,0 +1,258 @@
+from parameterized import parameterized
+
+from salt.exceptions import CommandExecutionError
+
+from salttesting.mixins import LoaderModuleMockMixin
+from salttesting.unit import TestCase
+from salttesting.mock import MagicMock, patch
+from salttesting.helpers import ForceImportErrorOn
+
+import metalk8s_etcd
+
+MEMBERS_LIST_DICT = [{
+    "client_urls": [
+        "https://10.11.12.13:2379"
+    ],
+    "id": "17971792102091431977L",
+    "name": "bootstrap",
+    "peer_urls": [
+        "https://10.11.12.13:2380"
+    ]
+}, {
+    "client_urls": [
+        "https://10.11.12.14:2379"
+    ],
+    "id": "17971792102091431978L",
+    "name": "node1",
+    "peer_urls": [
+        "https://10.11.12.14:2380",
+        "https://11.11.12.14:2380"
+    ]
+}]
+MEMBERS_LIST = [MagicMock(**(member)) for member in MEMBERS_LIST_DICT]
```

Comment:
details, superfluous parenthesis:
```suggestion
MEMBERS_LIST = [MagicMock(**member) for member in MEMBERS_LIST_DICT]
```

Reply:
Right i will change it